extensions/sql-migration/src/api/utils.ts

```diff
@@ -1348,3 +1428,29 @@ export async function promptUserForFile(filters: { [name: string]: string[] }):
 
 	return '';
 }
+export async function refreshIntegrationRuntimeTable(_view: ModelView, _integrationRuntimeTable: DeclarativeTableComponent,
+	migrationServiceMonitoringStatus: azure.IntegrationRuntimeMonitoringData): Promise<void> {
+	if (migrationServiceMonitoringStatus.nodes.length === 0) {
+		const data: DeclarativeTableCellValue[][] = [
+			[
+				{ value: " " },
+				{ value: "No node found" },
```

Comment:
same here, should be added as resource string in strings.ts

Reply:
Done